### DISCUSSION TITLE
How can i generate custom ores in the end

### DISCUSSION THREAD

**Discussion by Person0z** [DerFrZocker/Custom-Ore-Generator#30] [Q&A]
So as the title states, I want to generate ores on the bottom of the islands in the end. It's not the main island, and I want the ore to be rare and only spawn 1 ore block total, no veins or anything. How exactly would I do this? Are there youtube videos?

**Comment by DerFrZocker** [comment]
For a single vein you can use the single ore generator, to make it rare use a veins per chunk value around 1. To make it generated at the bottom use a low height range value. Finally to make it not generated on the main island, set it to generated in the Small End Islands, End Midlands, End Highlands and End Barrens biomes.

Tip: Begin with a height veins per chunk value ~20 and work your way down to the rarity you want, this way you can see if it does generated at all.
